community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/birk/il/Instruction.scala

```diff
@@ -58,5 +64,7 @@ object Instruction {
     override def fields(): String = ""
 
     override def generateInit(): String = ""
+
+    override def exceptions: Set[ExceptionCodeGen] = Set.empty
```

Comment:
It should override the underscore version, shouldn't it? I understand that `f` is the aggregating function and `_f` is the "local" function, is this correctly interpreted?